.github/workflows/deploy.yaml

```diff
@@ -33,19 +37,19 @@ jobs:
       - name: Download ${{ matrix.target }}/amd64 artifact
         uses: actions/download-artifact@v4
         with:
-          name: ansible-runner-${{ github.sha }}-${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-amd64
-          path: /tmp
+          name: ansible-runner-${{ github.sha }}-${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-amd64${{ inputs.target_tag_postfix }}${{ inputs.artifact_suffix }}
+          path: '/tmp'
 
       - name: Download ${{ matrix.target }}/arm64 artifact
         uses: actions/download-artifact@v4
         with:
-          name: ansible-runner-${{ github.sha }}-${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-arm64
-          path: /tmp
+          name: ansible-runner-${{ github.sha }}-${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-arm64${{ inputs.target_tag_postfix }}${{ inputs.artifact_suffix }}
+          path: '/tmp'
 
       - name: Load image
         run: |
-          docker load --input /tmp/${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-amd64.tar
-          docker load --input /tmp/${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-arm64.tar
+          docker load --input ${{ inputs.download_image_path }}/${{ matrix.versions.ansible }}-${{ matrix.target }}-linux-amd64.tar
```

Comment:
`download_image_path` is not defined in this workflow input. And if you want to make it a parameter, we should also maybe use it in the previous steps when downloading artifacts